libs/core---samd/pxt.json

```diff
@@ -14,6 +14,7 @@
         "switch": "file:../switch",
         "touch": "file:../touch",
         "thermometer": "file:../thermometer",
-        "lcd": "file:../lcd"
+        "lcd": "file:../lcd",
+        "mixer": "file:../mixer---samd"
```

Comment:
this should be `"mixer---samd": "file:../mixer---samd"`, otherwise it may get messed up when bundled (and replaced by `"*"`); same below.

Reply:
also, maybe we could support `"foo": "file:"` expanding to `"foo": "file:../foo"` to avoid this sort of problem in future